pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1398,7 +1397,7 @@ var _ = Describe("Converter", func() {
 		It("should handle float memory", func() {
 			vmi.Spec.Domain.Resources.Limits[k8sv1.ResourceMemory] = resource.MustParse("2222222200m")
 			xml := vmiToDomainXML(vmi, c)
-			Expect(strings.Contains(xml, `<memory unit="b">2222222</memory>`)).To(BeTrue(), xml)
+			Expect(xml).To(ContainSubstring(`<memory unit="b">2222222</memory>`))
```

Comment:
Will this produce the same output as before? Read will it print the whole xml?

Reply:
Good catch. By default, the strings in error messages for expected objects are truncated to 4000 characters, but the strings in descriptions are not limited.

I will increase the limit.

Reply:
Done